src/transform/convert/convertRecord.js

```diff
@@ -43,7 +41,10 @@ import {generate776} from './generate7xxFields';
 import {generate856} from './generate8xxFields';
 import {generateSID} from './generateSidFields';
 
-export default (fieldValueInterface, harvestSource) => {
+export default (fieldValueInterface, options) => {
+  const {harvestSource, sourceMap} = options;
+  const momentSource = options.moment || moment;
+
   const marcRecord = new MarcRecord();
   marcRecord.leader = '01704nam a22002653i 4500'; // eslint-disable-line functional/immutable-data
```

Comment:
1) Are records always text monographs 'am' - or are these leader values updated later if needed? 
2) Character positions 00-04 - Record length and 12-16 - Base address of data could be all zeroes, as actual values for these are not known yet.

Reply:
1. Good point. At this point I believe this is what the transformation supports, but we need to add filter to make it explicit.
2. Fixed